tools/deploy_assisted_installer_configmap.py

```diff
@@ -97,5 +101,14 @@ def main():
     )
 
 
+def get_admin_users():
+    admins_file = os.path.join(os.getcwd(), 'ADMINS')
```

Comment:
Can we set the admins on [OWNERS_ALIASES](https://github.com/openshift/assisted-service/blob/master/OWNERS_ALIASES) ?

Reply:
I don't think we should combine these

Reply:
I thought these are the same usernames. ignore